src/components/Images.vue

```diff
@@ -25,16 +25,17 @@
 		:mime="mime"
 		:src="src"
 		:fileid="fileid"
+		@updated="onUpdate"
 		@close="onClose" />
 
-	<img v-else
+	<img v-else-if="data !== null"
 		:alt="alt"
 		:class="{
 			dragging,
 			loaded,
 			zoomed: zoomRatio !== 1
 		}"
-		:src="data"
+		:src="`${data}${cacheBuster}`"
```

Comment:
Simple approach, but it works. Sexier idea welcome. :)

This makes sure that the initial request can use the browser's cache, and that on an update the cache is busted with `?buster=1`